bears/general/FilenameBear.py

```diff
@@ -63,6 +71,8 @@ def run(self, filename, file,
                           'Using the default "snake" naming convention.')
                 file_naming_convention = 'snake'
 
+        messages = []
```

Comment:
If we support merging diffs on a file-name basis, we can yield again different results for those :+1: